test-app-legacy-context/app/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "drizzle": "^1.2.4",
     "drizzle-react": "^1.2.0",
-    "drizzle-react-components": "^1.3.0",
+    "drizzle-react-components": "file:../..",
```

Comment:
I'd prefer to keep this at `^1.3.0`. It keeps things easier for the user to check out.

If it becomes a local npm dependency, the user has to `npm install` in 3 places (test-app, test-app/app, DRC root), and then `npm run build`.

If it's just an npm version the user can just do `npm install` in 2 places and that's it.

If people want to contribute they can modify this themselves locally, or use `npm link`. But I think improvements on the contributing process is out of scope for this PR. If you would like to improve the flow, please open a new issue and PR.